src/action.ml

```diff
@@ -271,21 +271,14 @@ module Unresolved = struct
 end
 
 module Var_expansion = struct
-  module Concat_or_split = struct
-    type t =
-      | Concat (* default *)
-      | Split  (* the variable is a "split" list of items *)
-  end
-
-  open Concat_or_split
-
   type t =
-    | Paths   of Path.t list * Concat_or_split.t
-    | Strings of string list * Concat_or_split.t
+    | Paths   of Path.t list
+    | Strings of string list
 
   let is_multivalued = function
-    | Paths (_, Split) | Strings (_, Split) -> true
-    | Paths (_, Concat) | Strings (_, Concat) -> false
+    | Paths [_] -> false
+    | Strings [_] -> false
+    | _ -> false
```

Comment:
shouldn't this be `true`?

Reply:
Hmm, why is that? If we only have 1 value then we aren't multivalued.

Reply:
Currently this function is the same as this one:

```
let is_multivalued _ = false
```

Reply:
Ah yes. Oops.